x-pack/plugins/security_solution/public/value_list/translations.ts

```diff
@@ -131,7 +124,7 @@ export const FAILED_TO_UPLOAD_LIST_ITEM_TITLE = i18n.translate(
 );
 
 export const UPLOAD_TOOLTIP = i18n.translate('xpack.securitySolution.listItems.uploadTooltip', {
-  defaultMessage: 'All items from the file will be added as new items',
+  defaultMessage: 'All items will be added to the value list',
```

Comment:
Minor punctuation edit here. 

```suggestion
  defaultMessage: 'All items will be added to the value list.',
```

@approksiu, does this revision look good to you, or do we need to tweak it further?

Reply:
Thanks @nastasha-solomon, maybe add "from the file" -> "All items from the file will be added to the value list."?

Reply:
Yeah, that works! I'll add the suggestion to this PR. Thanks, @approksiu